pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -340,9 +345,11 @@ func TestRecreateCompletedWorkflow(t *testing.T) {
 func TestInternalActorsSetupForWF(t *testing.T) {
 	ctx := context.Background()
 	_, engine := startEngine(ctx, t, task.NewTaskRegistry())
-	assert.Len(t, engine.GetInternalActorsMap(), 2)
-	assert.Contains(t, engine.GetInternalActorsMap(), workflowActorType)
-	assert.Contains(t, engine.GetInternalActorsMap(), activityActorType)
+	if abe, ok := engine.Backend.(*wfengine.ActorBackend); ok {
```

Comment:
I think we need to assert/require that the cast succeeded since the test is otherwise invalid.